### PR TITLE
Conda integration

### PR DESCRIPTION
**Pull request by FrodePedersen**
As discussed, we are working on a simple way of displaying whether a given tool exists on Conda, and if it does, how to easily install it through a command. As such we'd like to add the following changes:

1) Add a visual representation of the conda installation process (the command) bounded by a box on the resource page. That is, we would like to modify the frontend/components/toolpage/toolpage.html file to include this.

2) Add entries within the model, storing whether a recipe exists of the tool on conda, i.e. add a key storing the conda channel of the tool and one that stores the conda package name. As such, we would like to modify the backend/elixir/model/resource_model/resource.py to store these two new keys.

### CODE REVIEW

File: frontend/components/toolPage/toolPageConda.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div style="background:#eee; padding:1pc; border-radius:5px">
+    to install with <a href="https://conda.io/docs/user-guide/tasks/manage-pkgs.html">Conda:</a>
+        <div style="margin-left:2.5pc; margin-top:1pc"><code style="color:#666">conda install -c {{ software.conda_channel }} {{ software.conda_package }}</code></div>
```
Comment (bgruening) [inline frontend/components/toolPage/toolPageConda.html]: I like that! But what about channel dependencies? Do install a bioconda pkgs you also need to have conda-forge activated. Should conda_channel be a ordered list instead?

**Comment by joncison** [comment]
Thanks @FrodePedersen - let's discuss with @piotrgithub1 at the conference this week (if you're going)

**Comment by FrodePedersen** [comment]
Unfortunately, due to the very short notice we were given, neither @MathiasHaudgaard nor I can attend the meeting tomorrow. However, @dansondergaard will :). Looking forward to hear what may come of it.

File: frontend/components/toolPage/toolPageConda.html (hunk 1 above)
Comment (dansondergaard) [reply frontend/components/toolPage/toolPageConda.html]: Good call! Could just be a comma-separated list of channel names, and the installation instructions could then show how to add those channels.

**Comment by dansondergaard** [comment]
As discussed yesterday, we won't continue work on this PR right now, since some big changes to the codebase are coming up during next week. We'll then merge our changes with the new code and address @bgruening's comment.

**Pull request closed without merging**